python/analysis.i

```diff
@@ -0,0 +1,58 @@
+%module analysis
+
+%{
+#define SWIG_FILE_WITH_INIT
+#include <lttoolbox/fst_processor.h>
+#include <lttoolbox/lttoolbox_config.h>
+#include <lttoolbox/my_stdio.h>
+#include <lttoolbox/lt_locale.h>
+
+class FST: private FSTProcessor
+{
+public:
+  /**
+   * Reads from input_path and stores result at output_path
+   */  
+  void init_analysis(char *automorf_path, char *input_path, char *output_path);
+  bool validity() const;
+
+};
+
+bool 
+FST::validity() const
```

Comment:
Why can't the Python just directly call `valid()`?

Reply:
- The class is being inherited privately, to restrict access to other member functions from python
- Due to this validity() is used to call FSTProcessor::valid()

Reply:
> The class is being inherited privately, to restrict access to other member functions from python

Why? I don't really care right now how the surface of the wrapper looks (not sure I ever will?).